errors.js

```diff
@@ -0,0 +1,12 @@
+exports.handlePostgresErrors = (err, req, res, next) => {
+  next(err)
+};
+
+exports.handleCustomErrors = (err, req, res, next) => {
+  next(err);
+};
```

Comment:
Good that you're thinking ahead, but as these error handlers are currently not doing anything, can they be removed until later please? 😄

Reply:
removed postres and custom handler